src/crimson/os/seastore/transaction_manager.h

```diff
@@ -775,6 +775,7 @@ class TransactionManager : public ExtentCallbackInterface {
       remap_length,
       original_laddr,
       std::move(original_bptr));
+    ext->set_laddr(remap_laddr);
     return lba_manager->alloc_extent(
```

Comment:
seems laddr has been set when alloc remapped extent in lba tree, see https://github.com/ceph/ceph/blob/0857faed50c6de30d4709606de0c5c8c2dc77446/src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc#L282